src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml

```diff
@@ -40,8 +40,8 @@
     # delete any empty directories so they don't just build up (to literally 100,000+)
     # but leaving any that have been modified in the last 20m
     # (to avoid a race condition on newly created dirs)
-    # Also never delete {{ formplayer_access_log_dir }}
-    job: 'find {{ formplayer_data_dir }} -path {{ formplayer_access_log_dir }} -prune -o -empty -type d -mmin +20 -delete'
+    # Also never delete {{ formplayer_access_log_dir }} or {{ formplayer_data_dir }}
+    job: 'find {{ formplayer_data_dir }} -mindepth 1 -not -path {{ formplayer_access_log_dir }} -empty -type d -mmin +20 -delete'
```

Comment:
New suggested fix:

```suggestion
    job: "find {{ formplayer_data_dir }} -mindepth 1 -not -path {{ formplayer_access_log_dir }} -not -path '{{ formplayer_access_log_dir }}/*' -empty -type d -mmin +20 -delete"
```

Reply:
Hi Daniel, the reason I didn't bother excluding what's underneath the access log dir is that I assumed the point was to spare the access log directory itself, and there weren't going to be (empty) directories underneath it anyway. If that's not the case then, yes, that extra exclusion is needed.

Reply:
I'll patch in your change in another commit.

Another thing: I was looking for the actual cron file in `/etc/cron.*` but couldn't find it anywhere. Then remembered the old days, and indeed: it is in `/var/spool/crontab/cchq`, I suppose because its ansible task lacks the `cron_file` attribute.

I considered adding a patch to add the attribute (for consistency with the other jobs - this is the single one that uses classic crontab) but I suppose we'll then end up with both jobs when deploying on an existent system?

Reply:
Talking about cleanup: neither this cron job nor the one below is in `formplayer/tasks/cleanup.yml`, whereas the cron jobs above it are.